src/test/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/UserRoleControllerIntTest.kt

```diff
@@ -5,29 +5,36 @@ import org.junit.jupiter.api.Nested
 import org.junit.jupiter.api.Test
 import org.springframework.http.HttpStatus
 import org.springframework.http.MediaType
+import org.springframework.web.reactive.function.BodyInserters
 import uk.gov.justice.digital.hmpps.externalusersapi.integration.IntegrationTestBase
 
 class UserRoleControllerIntTest : IntegrationTestBase() {
 
+  val authAdmId = "5105A589-75B3-4CA0-9433-B96228C1C8F3"
```

Comment:
is auth relevant here?

Reply:
It's the name of the user in the test data